tests/training/test_transformation.py

```diff
@@ -524,19 +524,21 @@ def test_ecdfer():
     ecdf_column = "prediction_ecdf"
     max_range = 1000
 
-    pred_fn, data, log = ecdfer(fit_df, ascending, prediction_column, ecdf_column, max_range)
+    pred_fn, data, log = ecdfer(fit_df, ascending, prediction_column, ecdf_column, max_range, store_mapping=False)
```

Comment:
Parametrize this test with the value of `store_mapping` and ajust the `assert` statement to:

```python
assert (log["ecdfer"].get("map") is not None) == store_mapping
```